tools/processor.js

```diff
@@ -140,6 +161,15 @@ const Processor = new Lang.Class({
 
                 output['articles'].push(this._create_video(path, name, mime_type, category, this._sequence));
 
+            } else if (mime_type.startsWith('audio')) {
+                this._sequence += 1;
+
+                let category = null;
```

Comment:
Yes, it's in my list of refactors I want to make, I just wanted to have these basics in first.